data/campaigns/Under_the_Burning_Suns/scenarios/05_A_Subterranean_Struggle.cfg

```diff
@@ -698,11 +698,11 @@
     # When player approaches large cavern, a dying dwarf runs out and yells a warning.
```

Comment:
`[cancel_action]` would make sense here too, and maybe also for the stalactite crushing the spider

Reply:
Technically there are many more places where this could make sense, perhaps it would be better to do that in another PR.

Reply:
It seems logical to have it in the same commit in the history, as testing the `[cancel_action]` also tests the change of trigger.

Looking at this further, the kill-spider and meet-dwarf events are now effectively one big event (same event name, same filter) split across two `[event]` tags; the comments in the code should point that out for future readers.

Reply:
I actually haven't tested this part. Is there a point in keeping it as two events?

I'll check this out when I find the time.

Reply:
I think merging them into one makes sense.

Reply:
@soliton- will you have time before this weekend's release? I'm wondering about picking up the task myself.